app/src/main/java/aloksharma/ufl/edu/stash/ServerAccess.java

```diff
@@ -4,8 +4,16 @@
 import android.content.Intent;
 import android.support.v4.content.LocalBroadcastManager;
 import android.util.Log;
+import android.widget.ArrayAdapter;
```

Comment:
You have some unused imports lying around. Always before committing, make sure you remove all unused imports. Ctrl + Alt + O will automatically remove unused imports.